client/configuration.ini.sample

```diff
@@ -6,6 +6,8 @@ mcs-relay-url = http://staging.estcube.eu:8029/ax.25
 # Type of the SIDS relay request. Possible values are GET and POST.
 # (Currently only POST is supported by ESTCube servers).
 relay-request-type = POST
+#
+save-unrelayed-packets = True
```

Comment:
Why is this an option in the .ini file? It should do it by default anyways.

Reply:
Secondly, there is no comments about what this enables or disables.

Reply:
Removed the option from the configuration - it is now always done by default.